oozie-to-airflow/tests/mappers/test_pig_mapper.py

```diff
@@ -18,6 +18,7 @@
 from airflow.utils.trigger_rule import TriggerRule
 
 from mappers import pig_mapper
+from tests.utils.test_paths import EXAMPLE_PIG_PATH
```

Comment:
Is this used anywhere?

Reply:
Good point. I refactored it few hours ago :). Not needed. Will remove.